src/main/java/org/primeframework/mvc/action/DefaultActionMappingWorkflow.java

```diff
@@ -98,31 +97,46 @@ public void perform(WorkflowChain chain) throws IOException {
       throw new NotAllowedException();
     }
 
-    // Start the timer and grab a meter for errors
-    Timer.Context timer = null;
-    Meter errorMeter = null;
+    // Start the timers and grab some meters for errors
+    Timer.Context perPathTimer = null;
+    Timer.Context aggregateTimer = null;
+    Meter perPathErrorMeter = null;
+    Meter aggregateErrorMeter = null;
 
     try {
       if (metricRegistry != null && actionInvocation.action != null) {
         // Ignore try/resource inspection, we are closing this in the finally block already
+
+        //noinspection resource
+        perPathTimer = metricRegistry.timer("prime-mvc.[" + actionInvocation.uri() + "].requests").time();
+
         //noinspection resource
-        timer = metricRegistry.timer("prime-mvc.[" + actionInvocation.uri() + "].requests").time();
-        errorMeter = metricRegistry.meter("prime-mvc.[" + actionInvocation.uri() + "].errors");
+        aggregateTimer = metricRegistry.timer("prime-mvc.all.requests").time();
```

Comment:
Another option for naming would be `prime-mvc.[*].requests` and `prime-mvc.[*].errors`  - I don't mind `all` either - but the `[*]` would follow the current convention better perhaps?

Do you have any preference between these two options?

Reply:
That works fine for `/api/status`, but the prometheus exporter converts all of the special characters to underscores, so the resulting metric ends up looking like:
```
# HELP prime_mvc_____errors_total Generated from Dropwizard metric import (metric=prime-mvc.[*].errors, type=com.codahale.metrics.Meter)
# TYPE prime_mvc_____errors_total counter
prime_mvc_____errors_total 0.0
```

Which isn't great, but maybe it is better to have consistency in `/api/status` and then have a future iteration that cleans up the prom metrics?